.github/workflows/create-production-build.yml

```diff
@@ -0,0 +1,59 @@
+name: Build on Release Branch Merged into Deploy
```

Comment:
I am not sure about this name...Maybe consider either create production build (like the file) or Build Production on Release Merge to Deploy

Reply:
what about: Dispatch Production Build when Release Branch Merged into Deploy

Reply:
Create Production Build?

Reply:
https://github.com/digidem/comapeo-mobile/pull/618/commits/639ebaa8f20c98555a773dd657a91be32b2e8953